lib/paginate_firestore.dart

```diff
@@ -252,6 +256,54 @@ class _PaginateFirestoreState extends State<PaginateFirestore> {
 
     return listView;
   }
+
+  Widget _buildPageView(PaginationLoaded loadedState) {
+    var pageView = CustomScrollView(
+      reverse: widget.reverse,
+      shrinkWrap: widget.shrinkWrap,
+      controller: widget.scrollController,
+      scrollDirection: widget.scrollDirection,
+      physics: widget.physics,
+      slivers: <Widget>[
+        if (widget.header != null) SliverToBoxAdapter(child: widget.header),
+        SliverPadding(
+            padding: widget.padding,
+            sliver: SliverFillRemaining(
+              child: PreloadPageView.custom(
+                scrollDirection: Axis.vertical,
+                controller: widget.pageController,
+                childrenDelegate: SliverChildBuilderDelegate(
+                  (context, index) {
+                    if (index >= loadedState.documentSnapshots.length) {
+                      _cubit!.fetchPaginatedList();
+                      return widget.bottomLoader;
+                    }
+                    return widget.itemBuilder(
+                        index, context, loadedState.documentSnapshots[index]);
+                  },
+                  childCount: loadedState.hasReachedEnd
+                      ? loadedState.documentSnapshots.length
+                      : loadedState.documentSnapshots.length + 1,
+                ),
+              ),
+            )),
+        if (widget.footer != null) SliverToBoxAdapter(child: widget.footer),
```

Comment:
I think SliverToBoxAdapter is not necessary as we changed it to sliver by default in the last version.

Reply:
Agree